decidim-meetings/app/commands/decidim/meetings/admin/copy_meeting.rb

```diff
@@ -54,6 +54,9 @@ def copy_meeting!
             location_hints: @form.location_hints,
             services: @form.services_to_persist.map { |service| { "title" => service.title, "description" => service.description } },
             component: @meeting.component,
+            is_private: @form.is_private,
```

Comment:
Could you please you `private` and `transparent`? See https://github.com/bbatsov/ruby-style-guide#bool-methods-prefix (not sure why Rubocop doesn't complain)

Reply:
but `private` is a reserved key, no? so maybe we can use `private_meeting`. What you think?

Reply:
I'd say `private` is a valid method name... I can't think of any ambiguity from using it.

Reply:
That is a valid style remark, but in my opinion is ok. If the column name is an adjective, boolean is the only type that makes sense, no? And it's also the style used with other boolean columns. Basically we never use `is_` prefixes.

Reply:
Actually I just read https://stackoverflow.com/questions/16099073/ruby-reserved-class-method-names and it looks like @isaacmg410 is right and we shouldn't do this... For example, if you define a `private` class method, it makes the `private` keyword lose its meaning and instead calls the class method... So sorry for the extra friction here Isaac but I think it's probably safer to go back to your original version?

Reply:
private_meeting works for me!

Reply:
`private_meeting` looks perfect to me as well! Sorry again @isaacmg410 for the back and forth.